src/App.scss

```diff
@@ -30,3 +30,33 @@
     transform: rotate(360deg);
   }
 }
+.btn-primary{
+  padding: 0.5rem 2rem;
+  border: 1px white solid;
+  display: inline-block;
+  border-radius: 5px;
+  box-shadow: 0 9px #999;
+  height: auto;
+  cursor: pointer;
+  transition: all 0.5s;
+  text-decoration: none;
+  background: teal;
+  color: white;
+  //transition: background 500ms ease;
+}
+
+.btn-secondary{
+  padding: 0.5rem 2rem;
+  border: 1px teal solid;
```

Comment:
Use hexadecimal instead of literals for colors.
Apply this everywhere you've used literals